index.bs

```diff
@@ -1334,18 +1335,27 @@ Removing AB or TAG members</h5>
 
 	The [=AB=] or [=TAG=] <em class=rfc2119>must</em> hold a hearing
 	on the potential [=removal=] of a participant
-	if requested by at least three of the participants in the group.
+	if requested by at least three of the participants in the group
+	or if requested by a [=Team decision=]
```

Comment:
Being a Team decision makes it susceptible to FO, and thus to Councils. It's far from obvious to me that making that invokeable here is appropriate or useful.

My sense is that the goal here is to try and go for a de-personified CEO, so it comes from the CEO directly, or from the CEO via someone else, or from the Team collectively or an individual in the Team as longs as it's according to some delegation process endorsed by / established by the CEO and subject to their oversight. And simultaneously, that highlights that it's coming from the CEO as leader of the Team, not just from an individual who happens to be the CEO.

That seems fine. But activating the FO machinery seems like an undesirable side effect.